packages/web/src/react.test.tsx

```diff
@@ -46,4 +46,83 @@ describe('<Analytics />', () => {
       expect(script).toHaveAttribute('defer');
     });
   });
+
+  describe('track  custom events', () => {
+    beforeEach(() => {
+      // reset the internal queue before every test
+      window.vaq = [];
+    });
+    describe('queue custom events', () => {
+      it('event name only', () => {
+        render(<Analytics mode="production" />);
+
+        track('my event');
+
+        expect(window.vaq).toBeDefined();
+
+        if (!window.vaq) throw new Error('window.vaq is not defined');
+
+        expect(window.vaq[0]).toEqual([
+          'track',
+          {
+            name: 'my event',
+          },
+        ]);
+      });
+      it('with custom data', () => {
```

Comment:
```suggestion
      });
      
      it('with custom data', () => {
```

Reply:
`it` should define the behaviour too, `it('should allow custom data to be tracked', () => {` maybe?